src/mui/form/FormTab.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import FormField from './FormField';
 
-const FormTab = ({ label, icon, children, ...rest }) =>
+const FormTab = (
+    { label, icon, children, ...rest } // eslint-disable-line no-unused-vars
```

Comment:
Can be configured like this to allow using rest siblings as a way to filter items:

```js
"no-unused-vars": ["error", { "ignoreRestSiblings": true }]
```